docs/pipelines/multi-account/index.md

```diff
@@ -1,11 +1,46 @@
-# Deploying Multi-Account Pipelines
+# Multiple Infrastructure-Live Repos
 
-<!-- Placeholder for future content for Pipelines v2 -->
+We recommend using a single `infrastructure-live` repository for managing your organization's infrastructure.
+Sometimes, this isn't possible due to team structure, security requirements, or other limitations.
+In order to accommodate multiple infrastructure repositories, Gruntwork Pipelines is configurable.
+
+## Create Additional Repos
+
+New `infrastructure-live` repositories can be created using the same process described in the
+[Hello World](../hello-world#setting-up-the-repositories) documentation.
+
+:::info
+Once the repo is created, you'll need to set up machine user access using either the existing machine user and `PIPELINES_DISPATCH` PAT token,
+or one created specifically for this purpose. See [Machine Users](../using-pipelines/machine-users) for more information.
+:::
+
+No special configuration is required for the new `infrastructure-live` repository,
+the Pipelines Dispatch job will identify the source repository and pass that information
+to the shared `infrastructure-pipelines` repository.
+
+## Enable Additional Repos
+
+To ensure no unauthorized access is granted to your `infrastructure-pipelines` repository,
+an allowlist of `infrastructure-live` repositories exists at the root directory of the `infrastructure-pipelines` repository.
+To allow resources to be deployed by your new repository,
+add the repository to `repo-allowlist.txt` on a new line.
```

Comment:
Hm, we should probably have a single folder like `.gruntwork` that contains all config. Otherwise, we could have a smattering of files in the top-level repo over time.

I realize that the infra-pipelines repo is designed not to have more than a handful of files in the root directory, but my brain expects that `.something` folders in GitHub represent configuration. In fact, this is isomorphic with `.github`.

As far as using a TXT file with one line per entry, could we introduce a minimal config file (`.gruntwork/config.yml`?) here instead that we can keep adding to over time? Maybe something like this?

```yaml
# The git repos that have permissions to invoke Pipelines jobs
- repo-allowlist:
  - acme/team-1-infrastructure-live
  - acme/team-2-infrastructure-live
```